.github/workflows/packages-bin.yml

```diff
@@ -226,6 +231,17 @@ jobs:
           Set-Service $PgSvc.Name -StartupType manual
           $PgSvc.Start()
 
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const path = require('path')
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+            core.exportVariable('VCPKG_ROOT', process.env.VCPKG_INSTALLATION_ROOT || '');
```

Comment:
Question: why to export `VCPKG_ROOT` as an alias for `VCPKG_INSTALLATION_ROOT`? AFAICS `VCPKG_ROOT` is only used by the pg_config stub. Are there other uses? If not, this can be dropped and the pg_config stub can use `VCPKG_INSTALLATION_ROOT`.

Reply:
Thank you for this observation. However this means that, for people who would try to build psycopg_c using vcpkg, this script would fail if we use `VCPKG_INSTALLATION_ROOT`. If this is the case, your previous solution was better.

As I'm rewriting the script, I'll keep both the env vars into account.